src/settings.py

```diff
@@ -246,7 +246,7 @@ def _apply(
             ldap = _get_ldap_interface()
             ldap.update(
                 "cn=admins,ou=sudo",
-                {"sudoOption": ["!authenticate"] if passwordless_sudo else []},
+                {"sudoOption": "!authenticate" if passwordless_sudo else set()},
```

Comment:
Sooo does this mean now the code magically handle the case where the data is a string (and won't interpret a string as a list of char) ?